### DISCUSSION TITLE
联机功能投票

### DISCUSSION THREAD

**Discussion by bvkj-bvsdn** [Meloong-Git/PCL#5325] [提问] [answered]
官方敢出联机功能我就敢捐frp！

**Comment by tangge233** [comment, marked as answer]
具体事项在常见问题帮助中的“支持通过启动器联机游玩 Minecraft”可查看
- https://github.com/Hex-Dragon/PCL2/discussions/1930

**Reply by bvkj-bvsdn** [reply]
官方不打算出这个功能是吗？我的建议是出一个基于frpc的frp功能嵌套在pcl2里，然后可以捐frp那种，让用户自行选择frp节点，这样可以有效解决关于联机的相关问题

**Reply by tangge233** [reply]
> 官方不打算出这个功能是吗？

这个功能有计划的，但是过于敏感暂时不能上…

**Reply by Pigeon0v0** [reply]
> 官方不打算出这个功能是吗？我的建议是出一个基于frpc的frp功能嵌套在pcl2里，然后可以捐frp那种，让用户自行选择frp节点，这样可以有效解决关于联机的相关问题

之后会做，但是不会考虑 Frp 方案
